clients/drcachesim/reader/record_file_reader.h

```diff
@@ -149,7 +149,13 @@ class record_reader_t : public std::iterator<std::input_iterator_tag, trace_entr
         UNUSED(res);
```

Comment:
In the PR description: s/instrution/instruction/

Reply:
Done.